### ISSUE TITLE
Feature Request: Allow setting different column name for a column

### ISSUE THREAD

**Issue by abierbaum** [ayonious/console-table-printer#313]
I have objects I want to pass in as rows, but I don't want to use those keys as the column headers.  I would like to specify a column header name directly instead.

Something like:

```
const p = new Table({
  columns: [
    { name: "index", title: 'Index', alignment: "left", color: "yellow" }, // column coloring
    { name: "text", title: chalk.blue('Description'), alignment: "right" },
  ],
});
```
Is this possible already?

**Comment by ayonious** [comment]
Currently there is no way for this. But it's a good idea. I will work on it soon

**Comment by ayonious** [comment]
this is released in Version 2.7.

Sample code is (as mentioned/requested by @abierbaum)
```
const p = new Table({
  columns: [
    { name: "index", title: 'Index', alignment: "left", color: "yellow" }, // column coloring
    { name: "text", title: chalk.blue('Description'), alignment: "right" },
  ],
});
```

Note that this title will be used only for showing as header. For mapping on rows you still need to use name.

```
p.addRow({
  index: 1,
  text: 'something' // here using the name field or the column
})
```
